pwndbg/heap/ptmalloc.py

```diff
@@ -93,6 +125,18 @@ def malloc_alignment(self):
         return pwndbg.arch.ptrsize * 2
 
 
+    @property
+    @pwndbg.memoize.reset_on_objfile
+    def malloc_size_sz(self):
```

Comment:
If it corresponds to some define/constant within libc code, can we add some info about that in a docstring? (maybe even add a link to source)

Reply:
Same for the rest